crates/uv-workspace/src/pyproject.rs

```diff
@@ -113,6 +113,20 @@ impl AsRef<[u8]> for PyProjectToml {
     }
 }
 
+/// A specifier item in a PEP 735 Dependency Group
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case", untagged)]
+pub enum DependencyGroupSpecifier {
+    /// PEP 508 requirement string
+    Requirement(String),
+    /// Include another dependency group
+    #[serde(rename_all = "kebab-case")]
+    Table {
+        /// The name of the group to include
+        include_group: Option<String>,
```

Comment:
Should this be required?

Reply:
If you make this `Option<GroupName>`, it will simplify some error handling. You can probably then also avoid some `GroupName` clones (I think `parents` can become `BTreeSet<&GroupName>`).

Reply:
I had the same thought. I think it's written this way so other fields can be added here in the future then they can be made mutually exclusive or not based on whether or not they're populated? We should probably fail with an empty table though? Or at least warn after deserialization?